src/OrchardCore.Modules/OrchardCore.Metadata/Drivers/MetadataPartDisplayDriver.cs

```diff
@@ -0,0 +1,244 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OrchardCore.ContentManagement.Display.ContentDisplay;
+using OrchardCore.ContentManagement.Display.Models;
+using OrchardCore.ContentManagement.Metadata;
+using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.DisplayManagement.Views;
+using OrchardCore.Metadata.Models;
+using OrchardCore.Metadata.ViewModels;
+using OrchardCore.ResourceManagement;
+using OrchardCore.Settings;
+
+namespace OrchardCore.Metadata.Drivers
+{
+    public class MetadataPartDisplayDriver : ContentPartDisplayDriver<MetadataPart>
+    {
+        private readonly IContentDefinitionManager _contentDefinitionManager;
+        private readonly ISiteService _siteService;
+        private readonly IResourceManager _resourceManager;
+
+        public MetadataPartDisplayDriver(
+            IContentDefinitionManager contentDefinitionManager,
+            ISiteService siteService, IResourceManager resourceManager)
+        {
+            _contentDefinitionManager = contentDefinitionManager;
+            _siteService = siteService;
+            _resourceManager = resourceManager;
+        }
+
+        public override IDisplayResult Display(MetadataPart metadataPart)
+        {
+            return Initialize<MetadataPartViewModel>("MetadataPart", async model => await BuildViewModelAsync(model, metadataPart))
```

Comment:
No need for the async/await keywords here, as the only thing awaited is returned